modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/SpringCodegen.java

```diff
@@ -536,6 +536,8 @@ public void processOpts() {
 
         additionalProperties.put("lambdaSplitString", new SplitStringLambda());
 
+        // apiController: hide implementation behind undocumented flag to temporarily preserve code
+        additionalProperties.put("_api_controller_impl_", false);
```

Comment:
For reference - is there a discussion on when to remove this?

Reply:
Left as a fallback in case there are issues with the spring-mvc removal. Could be removed at the latest with 7.0.0 if not needed.